lib/ci-stack.ts

```diff
@@ -155,10 +155,9 @@ export class CIStack extends Stack {
 
       this.agentNodes = [
         agentNode.AL2023_X64, agentNode.AL2_X64_DOCKER_HOST, agentNode.AL2023_X64_DOCKER_HOST,
-        agentNode.AL2023_ARM64, agentNode.AL2_ARM64_DOCKER_HOST, agentNode.AL2023_ARM64_DOCKER_HOST,
-        agentNode.AL2023_X64_DOCKER_HOST_PERF_TEST, agentNode.AL2023_X64_DOCKER_HOST_BENCHMARK_TEST,
-        agentNode.UBUNTU2004_X64_GRADLE_CHECK, agentNode.UBUNTU2004_X64_GRADLE_CHECK_NEW_SPECS, agentNode.UBUNTU2004_X64_DOCKER_BUILDER,
-        agentNode.MACOS12_X64_MULTI_HOST, agentNode.WINDOWS2019_X64, agentNode.WINDOWS2019_X64_GRADLE_CHECK,
+        agentNode.AL2023_ARM64, agentNode.AL2_ARM64_DOCKER_HOST, agentNode.AL2023_ARM64_DOCKER_HOST, agentNode.AL2023_X64_DOCKER_HOST_BENCHMARK_TEST,
+        agentNode.UBUNTU2004_X64_GRADLE_CHECK, agentNode.UBUNTU2004_X64_DOCKER_BUILDER, agentNode.MACOS12_X64_MULTI_HOST,
+        agentNode.WINDOWS2019_X64, agentNode.WINDOWS2019_X64_DOCKER_HOST, agentNode.WINDOWS2019_X64_DOCKER_BUILDER, agentNode.WINDOWS2019_X64_GRADLE_CHECK,
```

Comment:
nit: Can you refactor it to put each agent on one line? Easy to maintain and list.
Thanks!

Reply:
Updated now, thanks.